cmd/nitro/nitro.go

```diff
@@ -230,7 +234,12 @@ func openInitializeChainDb(ctx context.Context, stack *node.Node, initConfig *In
 		if err != nil {
 			return nil, nil, fmt.Errorf("couln't open init '%v' archive: %w", initFile, err)
 		}
-		err = extract.Archive(context.Background(), reader, stack.InstanceDir(), nil)
+		stat, err := reader.Stat()
+		if err != nil {
+			return nil, nil, err
+		}
+		log.Info("extracting downloaded init archive", "size", fmt.Sprintf("%dMB", stat.Size()/1024/1024))
+		err = extract.Archive(ctx, reader, stack.InstanceDir(), nil)
```

Comment:
I'm reconsidering whether this should be cancellable actually. We don't want to leave users with a half-extracted db.

Reply:
I've decided to extract the files to a temporary directory, and then move them atomically once it's done